### ISSUE TITLE
Collecting feedback of mass conversion PXPs->NWBv2

### ISSUE THREAD

**Issue by t-b** [AllenInstitute/MIES#458]
Trial of converting 189 experiments:

- [x] Memory leak
- [x] Another memory leak is also work around now.
- [x] The following PXPS 
   - Pvalb-IRES-Cre;Ai14-204092.04.02.pxp
   - Pvalb-IRES-Cre;Ai14-212440.05.02.pxp
   - Sst-IRES-Cre;Ai14-212983.04.01.pxp

bug out with

```
      "output": "Please be patient while we export all existing acquired content of all devices to NWB\r  !!! Assertion FAILED !!!\r  Message: \"Labnotebook is too old for NWB export.\"\r  Please provide the following information if you contact the MIES developers:\r  ################################\r  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r  Stacktrace:\rAfterFileOpenHook(...)#L227 [MIES_MassExperimentProcessing.ipf]\rProcessCurrentExperiment(...)#L111 [MIES_MassExperimentProcessing.ipf]\rPerformMiesTasks(...)#L147 [MIES_MassExperimentProcessing.ipf]\rNWB_ExportAllData(...)#L406 [MIES_NeuroDataWithoutBorders.ipf]\rNWB_AppendSweepLowLevel(...)#L728 [MIES_NeuroDataWithoutBorders.ipf]\r  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r  Time: 2020-02-13T18:14:43-08:00\r  Locked device: [ITC18USB_Dev_0]\r  Current sweep: [75]\r  DAQ: [DAQ_NOT_RUNNING]\r  Testpulse: [TEST_PULSE_NOT_RUNNING]\r  Experiment: Pvalb-IRES-Cre;Ai14-204092.04.02.pxp\r  Igor Pro version: 8.0.5.1 (34845)\r  MIES version:\r  Release_2.0_20190602-699-g733a1754\rDate and time of last commit: 2020-02-13T16:44:29-08:00\rSubmodule status:\r 56cd64bfb2ceff2f2f074171d931c2a299951caf Packages/IPNWB (v0.19-23-g56cd64b)\r 9a3f02fb7d007b4b34b29bda1f3fddc0ce64ad33 Packages/ITCXOP2 (remotes/origin/old-master-1-g9a3f02f)\r aa1858d915ee28ebf5974f59f0bc19a421e5c28e Packages/ZeroMQ (heads/master)\r 7d239dad06bf51220cc63460373d21afcec0da91 Packages/doc/doxygen-filter-ipf (remotes/origin/old-master-1-g7d239da)\r a097151fbc9a8a8f2e504c9e46adcb327a19c8f3 Packages/unit-testing (UnitTestingFramework-v1.08-88-ga097151)\r  ################################\r  Caught an RTE
```

in all three cases sweep rollback was used. And as we did not have the entry source type back then, this breaks fetching any labnotebook entries.
- [x] FunctionPath does not work reliable if used in a IM. It also does not work if ProcGlobal is uncompiled.
- [x] Old style configuration notebook is missing

![image (1)](https://user-images.githubusercontent.com/1185677/76755371-08d8ee80-6784-11ea-8748-4bce094d7974.png)

- [ ] Nos1-CreERT2;Ai14-421031.03.01.01.pxp: Missing AD LBN entries in sweep 46. No clue why.
- [ ] H18.26.404.11.05.01.01.pxp: ??

**Comment by t-b** [comment]
Conversion of a batch of 387 PXPs worked (with the three PXPs with the labnotebook issues being the only problematic ones). No more OOM errors.

**Comment by t-b** [comment]
The labnotebook issue is fixed in #478.

**Comment by t-b** [comment]
Done.
